scripts/rook.sh

```diff
@@ -75,6 +84,12 @@ function teardown_rook() {
 	kubectl delete -f "${ROOK_URL}/toolbox.yaml"
 	kubectl delete -f "${ROOK_URL}/cluster-test.yaml"
 	kubectl delete -f "${ROOK_URL}/operator.yaml"
+	ROOK_MAJOR=$(rook_version 1)
```

Comment:
not sure we need to have an explicit delete based on the version, but thats fine too